src/main/resources/application.yml

```
@@ -0,0 +1,6 @@
+server:
+  port: 8080
+spring :
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)